ItemRack/ItemRack.lua

```diff
@@ -151,11 +151,31 @@ function ItemRack.OnLoad(self)
 end
 
 ItemRack.EventHandlers = {}
+ItemRack.ExternalEventHandlers = {}
 
 function ItemRack.OnEvent(self,event,...)
 	ItemRack.EventHandlers[event](self,event,...)
 end
 
+function ItemRack.OnItemRackEvent(self,event,handler)
```

Comment:
Please add a comment that explains how this is called and who is (currently) using it.
Also, I would like the name to indicate that this function is used to register and external event handler.

Reply:
What name would you like for the function?  That's the best I could come up with.  `RegisterExtenralEventListener`?  `AddExternalEventHandler`?

Reply:
RegisterExternalEventListener is fine